static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -159,9 +165,87 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
       this.searchThisArea();
     });
 
+    this.setupMapViewCssClasses();
     this.addMapComponent();
   }
 
+  /**
+   * Properly set CSS classes for mobile and desktop
+   */
+  setupMapViewCssClasses () {
+    if (!this.isMobile()) {
+      this.updateCssForDesktop();
+    }
+    window.addEventListener('resize', () => {
```

Comment:
I'm worried about this event being called a lot every time you resize. On Consulting we had a throttle based on animation frames for resize events to prevent it from being called too many times in one resize. Is that something we can do here?

Reply:
We may also be interested in the `addListener` for the media query object (so we can use both matches in the `isMobile` function and `addListener` for listening to breakpoint changes)

https://blog.logrocket.com/introduction-to-js-media-queries/